api/src/pcapi/repository/beneficiary_import_queries.py

```diff
@@ -25,6 +40,31 @@ def get_already_processed_applications_ids(procedure_id: int) -> set[int]:
     }
 
 
+def get_already_processed_applications_ids_from_fraud_checks(procedure_id: int) -> set[int]:
+    fraud_check_qs = fraud_models.BeneficiaryFraudCheck.query.filter(
```

Comment:
qs ?

Reply:
queryset :smiley: 
But if you have to ask... je renomme